src/components/characters.py

```diff
@@ -0,0 +1,122 @@
+import os
+from flask import abort, request, session
+from sqlalchemy import text
+from sqlalchemy.exc import IntegrityError
+
+from db.db import db
+
+
+def get_all_characters():
+    try:
+        sql = text("""
+            SELECT *
+            FROM characters
+        """)
+        result = db.session.execute(sql)
+        character_list = result.fetchall()
```

Comment:
I like this use of an extra variable in these. It makes it easier to understand since the fetchall/fetchone doesn't really say much.

All these functions are also very nice to read.

Reply:
I'm just coming from using heavy Kotlin so there is always an itch to write super long one-liners just for the "fun" of it. The "extra" variable is me trying to avoid this.